src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -195,31 +203,36 @@ CHIP_ERROR AdvertiserMinMdns::Start(chip::Inet::InetLayer * inetLayer, uint16_t
 /// Stops the advertiser.
 CHIP_ERROR AdvertiserMinMdns::StopPublishDevice()
 {
-    mQueryResponderAllocator.Clear();
+    mQueryResponderAllocatorOperational.Clear();
+    mQueryResponderAllocatorCommissionable.Clear();
+    mQueryResponderAllocatorCommissioner.Clear();
     return CHIP_NO_ERROR;
 }
 
 CHIP_ERROR AdvertiserMinMdns::Advertise(const OperationalAdvertisingParameters & params)
 {
-    mQueryResponderAllocator.Clear();
+    // TODO: When multi-admin is enabled, commissionable does not need to be cleared here.
+    mQueryResponderAllocatorOperational.Clear();
+    mQueryResponderAllocatorCommissionable.Clear();
```

Comment:
We do not want to clear these. Commissionable and Commissioner are independent of Operational
```suggestion
```